packages/shell-api/src/abstract-cursor.ts

```diff
@@ -63,7 +63,7 @@ export abstract class AbstractCursor extends ShellApiWithMongoClass {
 
   async _it(): Promise<CursorIterationResult> {
     const results = this._currentIterationResult = new CursorIterationResult();
-    await iterate(results, this, this._batchSize ?? await this._mongo._batchSize());
+    await iterate(results, this, await this._mongo._displayBatchSize());
```

Comment:
This means that the `batchSize` method here and the `_batchSize` property on the abstract cursor can be removed?

Reply:
Ah good spot - I think we can also refactor that a bit since we only have `AggregationCursor` and `Cursor` as the subclass which are essentially wrappers for driver cursors.